aws/aws-how-to/instances/upgrade-to-ubuntupro-with-license-manager.rst

```diff
@@ -0,0 +1,50 @@
+Upgrade Ubuntu instances to Ubuntu Pro with AWS License Manager
+===============================================================
+
+
+AWS released a new functionality that allows users upgrade Ubuntu LTS instances to `Ubuntu Pro`_ with `AWS License Manager`_.
+
+
+Requirements
+------------
+
+To upgrade to Ubuntu Pro, you will need
+
+* Ubuntu LTS EC2 machines 16.04 or later LTS versions
```

Comment:
replace `machine` with `instance`. that's the terminology AWS uses mostly afaik

Reply:
switched them